lib/app/dependency.go

```diff
@@ -130,6 +144,32 @@ func getDependencies(app *Application, apps Applications, state *state) error {
 	return nil
 }
 
+// AppsToExclude returns a list of apps that should be excluded
+func AppsToExclude(manifest schema.Manifest) []loc.Locator {
+	var excludeApps []loc.Locator
+	for _, app := range OptionalDependencies {
+		if schema.ShouldSkipApp(manifest, app) {
+			excludeApps = append(excludeApps, app)
+		}
+	}
+
+	return excludeApps
+}
+
+// AppsToExcludeFromManifest returns a list of dependencies that should be excluded
+func AppsToExcludeFromManifest(manifestPath string) ([]loc.Locator, error) {
+	if manifestPath != "" {
```

Comment:
This is a bit weird condition, can anything actually call this function with empty manifest path, and why?